src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -453,6 +478,17 @@ protected function getContextShopId()
         return $this->getContext()->shop->id;
     }
 
+    /**
+     * @param FormInterface $form
+     */
+    protected function addFlashFormErrors(FormInterface $form)
+    {
+        /** @var FormError $formError */
+        foreach ($form->getErrors() as $formError) {
```

Comment:
```suggestion
        foreach ($form->getErrors(true, false) as $formError) {
```
Maybe something more generic?  https://symfony.com/doc/current/components/form.html#accessing-form-errors

Reply:
I think `$form->getErrors(true)` is enough, so we have a one depth array with all the errors

Reply:
Good to me if we use at least the first parameter